README.Rmd

```diff
@@ -145,3 +145,10 @@ and if you choose to contribute, you must adhere to its terms.
 [emoji-print]: https://twitter.com/ptrckprry/status/887732831161425920 "MacOS Emoji Printing"
 [issues]: https://github.com/patperry/r-utf8/issues "Issues"
 [windows-enc2utf8]: https://twitter.com/ptrckprry/status/901494853758054401 "Windows enc2utf8 Bug"
+
+<!-- Session information -->
+<!-- ------- -->
+
+<!-- ```{r} -->
+<!-- sessioninfo::session_info(include_base = TRUE) -->
+<!-- ``` -->
```

Comment:
I have commented it out because the output format for `README.Rmd` is:

```yaml
---
output: downlit::readme_document
---
```

But there is no such function:
https://downlit.r-lib.org/dev/reference/index.html

Reply:
@krlmlr Lemme know if I am missing something obvious here.